apm-agent-plugins/apm-spring-resttemplate/apm-spring-restemplate-plugin/pom.xml

```diff
@@ -3,16 +3,16 @@
     <modelVersion>4.0.0</modelVersion>
 
     <parent>
-        <artifactId>apm-agent-plugins</artifactId>
         <groupId>co.elastic.apm</groupId>
+        <artifactId>apm-spring-resttemplate</artifactId>
         <version>1.19.1-SNAPSHOT</version>
     </parent>
 
     <artifactId>apm-spring-resttemplate-plugin</artifactId>
     <name>${project.groupId}:${project.artifactId}</name>
 
     <properties>
-        <apm-agent-parent.base.dir>${project.basedir}/../..</apm-agent-parent.base.dir>
+        <apm-agent-parent.base.dir>${project.basedir}/../../..</apm-agent-parent.base.dir>
     </properties>
 
```

Comment:
How can we make sure we're always testing against the latest spring version as they get released? Maybe use a version range here or update the spring version via dependabot?

Reply:
We can use [version ranges with Ivy](https://ant.apache.org/ivy/history/latest-milestone/ivyfile/dependency.html), I've changed the version 5.3.0 in the integration test to use that, and as a result it's currently testing with version 5.3.1.